### ISSUE TITLE
QA rules about waiting for confirmation should be revised

### ISSUE THREAD

**Issue by ypshenychka** [zerocracy/farm#724]
QA rules currently [say](http://www.zerocracy.com/policy.html#42):
>Until you get a confirmation, don’t give any quality mark.

Sometimes it gets to situation, when person doesn't respond on QA remarks. Reasons can be different: person is either not interested, or just not responsible, or is not on project anymore, etc.

And this situation leads to performer's resign due to 10 days rule, so evaluating this ticket as acceptable doesn't make sense anymore, because new performer didn't do anything as far as job is already done. Or the performer of a ticket is that person, but he doesn't work anymore and doesn't respond.

But QA has to wait due to the rule and as a result ticket will remain hanging and QA will be never be paid for it.

I suggest to add new idle period (e.g. also 10 days), to which QA can wait for confirmation. After 10 days QA can give a verdict (I think that it will be 'bad' then).

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@ypshenychka you are right, the current situation is not really logical. We will change it as soon this one is implemented: #646 We will close the order and the job right when it's closed. Then, we will create a QA review request and will wait for its completion. We will wait for as long as needed. When it's done, we pay for the job and to the QA inspector.

Thus, there will be no necessity to have any other X-days rules for the QA. You will be interested to give your feedback as soon as possible, in order to be paid.

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
com.amazonaws.http.AmazonHttpClient$RequestExecutor[1113] com.amazonaws.SdkClientException: Unable to execute HTTP request: null
sun.nio.ch.FileChannelImpl[110] java.nio.channels.ClosedChannelException: null
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm/tree/1.0-SNAPSHOT) /cc @yegor256
